app/components/simple-editor.js

```diff
@@ -92,6 +93,75 @@ import {
   text_variable,
   textVariableView,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/variable-plugin/variables';
+
+/**
+ * If the user config is present, merge it with the default config.
+ * Otherwise return the default config.
+ * @param defaultConfig
+ * @param userConfig
+ * @returns {*}
+ */
+const maybeMergeUserConfig = (defaultConfig, userConfig) => {
+  if (userConfig) {
+    return merge(defaultConfig, userConfig);
+  }
```

Comment:
Using merge is a good idea, but it has one problem: for lists is will concatenate the lists.
For our config usecase, this is actually not what we want.

Example:
The user adds
```
date {
 formats: [
    {
      label: 'Short Date',
      key: 'short',
      dateFormat: 'dd/MM/yy',
      dateTimeFormat: 'dd/MM/yy HH:mm',
    },
}
```

Now the date will have _three_ formats, instead of one, because the default formats will stay.